magento/pwa-studio
# [Doc] 6.0 release notes updates

## Description

Updates the changelog with additional content and adds an entry to the compatibility matrix.

## Related Issue

## Acceptance 

@dpatil-magento 

### Verification Stakeholders

@dpatil-magento 

### Specification

### Verification Steps
1. Open changelog
2. Verify entries are accurate

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
